source/_integrations/compit.markdown

```diff
@@ -0,0 +1,68 @@
+| BWC310        | Mixing valve controller    |                                                                                                                                                                               |
+| BioMax775     | Pellet boiler controller   |                                                                                                                                                                               |
+| BioMax772     | Pellet boiler controller   |                                                                                                                                                                               |
+| BioMax742     | Pellet boiler controller   |                                                                                                                                                                               |
+| SHC           | Carbon dioxide sensor      | [Product page](https://compit.pl/produkty/osprzet/67-czujnik-stezenia-dwutlenku-wegla-wilgotnosci-i-temperatury-w-pomieszczeniach-shc.html?ic=1)                              |
+| SPM           | Air quality sensor         | [Product page](https://compit.pl/produkty/osprzet/87-czujnik-jakosci-powietrza-spm.html?ic=1)                                                                                 |
+| L2            | Floor heating controller   | [Product page](https://compit.pl/produkty/sterowniki-ogrzewania-podlogowego/40-sterownik-ogrzewania-podlogowego-l2.html?ic=1)                                                 |
+| COMBO         | Combo module               | [Product page](https://compit.pl/produkty/osprzet/92-combo.html?ic=1)                                                                                                         |
+| EL750         | Electric boiler controller | [Product page](https://compit.pl/produkty/sterowniki-do-kotlow/73-sterownik-kotla-elektrycznego-el750-1.html?ic=1)                                                            |
+| R350.M        | Universal controller       | [Product page](https://compit.pl/produkty/sterowniki-uniwerslane/85-pogodowy-regulator-temperatury-obiegu-grzewczego-z-mieszaczem-r350m.html?ic=1)                            |
+| R350 T3       | Universal controller       | [Product page](https://compit.pl/produkty/sterowniki-instalacji/42-dwustopniowy-sterownik-temperatury-regulator-pi-regulator-krokowy-sterowanie-3-punktowe-r350-07.html?ic=1) |
+| R350.CWU      | Universal controller       | [Product page](https://compit.pl/produkty/sterowniki-uniwerslane/78-sterownik-do-podgrzewania-wody-r350-cwu.html?ic=1)                                                        |
+| AF-1          | Anti-freeze system         | [Product page](https://compit.pl/produkty/osprzet/91-af-1.html?ic=1)                                                                                                          |
+
+## Configuration in the UI
```

Comment:
please follow the format in https://developers.home-assistant.io/docs/core/integration-quality-scale/rules/docs-installation-instructions